### ISSUE TITLE
GitMob silentRun: "git mob --list" Cannot read property 'rootUri' of undefined (in a multi-repo workspace)

### ISSUE THREAD

**Issue by davidalpert** [rkotze/git-mob-vs-code#39]
### Prerequisites

- [x] [Checked that your issue isn't already filed][git-mob-vs-code issues]
- [x] Tried upgrading to the latest git-mob-vs-code and git-mob version (`npm i -g git-mob`)
- [x] Tried upgrading to the latest git version (`brew upgrade git` if installed with Homebrew/Linuxbrew)

[git-mob-vs-code issues]: https://github.com/rkotze/git-mob-vs-code/issues?utf8=%E2%9C%93&q=is%3Aissue

### Description

The following error is raised repeatedly while working in a VS Code workspace with multiple folders, each it's own git repo independent of the others:

```
GitMob silentRun: "git mob --list" Cannot read property 'rootUri' of undefined
```

### Steps to Reproduce

1. Install the `git-mob-vs-code` extension
2. Open a new VS Code window
3. Add three independent folders into the workspace, each of which is it's own git repo, independent of the others

**Expected behaviour:** [What you expect to happen]

- the git mob vscode extension works as expected, carrying my mob configuration into each repo as we work together in the workspace

**Actual behaviour:** [What actually happens]

- the git mob vscode extension works as expected, carrying my mob configuration into each repo as we work together in the workspace

**Reproduces how often:** [What percentage of the time does it reproduce?]

- every time I open the workspace
- again when I first switch to the `Source Control: Git` view

Once having clicked on one of the repos in the `Source Control Providers` panel I see that repo properly reflected in the `Git Mob Co-Authors` panel.

After one repo is selected, selecting other repos seems to work as expected, and I see each selected repo change reflect as expected in the `Git Mob Co-Authors` panel.

### Versions

- operating system and version: `macOS Catalina Version 10.15.5`
- git-mob-vs-code version: `1.5.3`
- git-mob version (`git-mob --version`): `1.1.0`
- git version (`git --version`): `git version 2.27.0` (installed via Brew and updated this morning)

### Additional Information

I suspect this error comes from [this line of code](https://github.com/rkotze/git-mob-vs-code/blob/v1.5.3/src/vscode-git-extension/git-ext.js#L20) when first opening VS Code and no repository has been selected in the `Source Control Providers` panel:

```csharp
  get rootPath() {
    if (this.repositories.length > 0)
      return this.selectedRepository.rootUri.fsPath;

    return "";
  }
```

Perhaps this method could be updated to return the first repository in `this.repositories` when `this.selectedRepository` is `null`?

If this bug is reproducible by others and my suggestion is valid I am happy to submit a PR.

**Comment by rkotze** [comment]
Hi @davidalpert thanks for the detailed issue, very helpful :)

Just to check the UI and the commands work **fine** but these error notification pop up when using workspaces?

**Comment by davidalpert** [comment]
that is correct yes.

**Comment by rkotze** [comment]
@davidalpert I've published a fix [v1.5.4](https://github.com/rkotze/git-mob-vs-code/releases/tag/v1.5.4) -> hopefully that solves it.

**Comment by davidalpert** [comment]
have not seen this repeat since upgrading to v1.5.4; thanks @rkotze
